src/reports.cpp

```diff
@@ -1573,10 +1573,9 @@ REPORT_GENERATOR(terrain, rc)
 			str << map.get_terrain_info(terrain).income_description();
 		} else if (owner == viewing_side) {
 			str << map.get_terrain_info(terrain).income_description_own();
-		} else if (viewing_team.is_enemy(owner)) {
-			str << map.get_terrain_info(terrain).income_description_enemy();
 		} else {
-			str << map.get_terrain_info(terrain).income_description_ally();
+			const team &viewing_team = rc.teams()[owner - 1];
+			str << VGETTEXT("$village_owner's village", {{ "village_owner", viewing_team.team_name() }});
```

Comment:
"viewing team" is the wrong name for this. In a multiplayer game, the viewing_team is the side of the player whose computer this wesnoth-client is running on, thus which side's shroud and fog should be applied.

Reply:
Fixed. Replaced it by `owner_team`